### PR TITLE
[GraphQL] configure expose exceptions from configuration Fixes #2879

### LINKED ISSUE #2879: [GraphQL] configure exception exposure via configuration not debug flag
At the moment, GraphQL swallows any exceptions that occur during graphql excecution, only exposing them if debugging. 

https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/GraphQLMiddleware.cs

image

This is fine when developing from source, but when using nu/myget debug isnt true i believe. It would be nice if we could set expose exceptions via configuration.

/cc @Jetski5822

### PR DESCRIPTION
**Pull request by Jetski5822**
Fixes #2879

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Startup.cs [hunk 1]
```diff
@@ -27,13 +35,20 @@ public override void ConfigureServices(IServiceCollection services)
 
         public override void Configure(IApplicationBuilder app, IRouteBuilder routes, IServiceProvider serviceProvider)
         {
+#if DEBUG
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Startup.cs]: Why not use IHostingEnvironment instead?

Comment (Jetski5822) [reply src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Startup.cs]: Good point!!! Forgot about that interface. I will make the change tomorrow.

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Startup.cs]: Use the value of `IsDevelopment` directly in the default value argument.

Comment (Jetski5822) [reply src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Startup.cs]: Sweet :)

**Pull request merged**